CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ This gem is moving onto its own [Semantic Versioning](https://semver.org/) schem
 
 Prior to v1.0.0 this gem was versioned based on the `MAJOR`.`MINOR` version of RuboCop. The first release of the ezcater_rubocop gem was `v0.49.0`.
 
+## v3.0.1
+- Exclude appraisal generated gemfiles for possible embedded gems in rails apps
+
```

Comment:
https://github.com/ezcater/ezcater_rubocop/pull/105/files is also introducing v3.0.1.  Do we wanna put both changes into this version or ask one of the two PRs to introduce v3.0.2 instead?

Reply:
I just merged mine. Do you want to merge the changelog and release the new version?